src/services/health-check/__tests__/get-health-check.integration.test.js

```diff
@@ -2,7 +2,7 @@ import { getHealthCheck } from '../get-health-check';
 import ModelFactory from '../../../models';
 import { config } from '../../../config';
 
-describe('getHealthCheck', () => {
+describe.skip('getHealthCheck', () => {
```

Comment:
Is this is being skipped, is it worth adding a deprecation note if it is going to be removed?

Reply:
this one will be un-skipped in PRMT-4587, so probably just leave it like this